README.md

```diff
@@ -12,13 +12,11 @@ For frontend, see [vets-website](https://github.com/department-of-veterans-affai
 
 ## Base setup
 
-**See the [native setup instructions](docs/setup/native.md) if you can't use docker**
+Developers who work with vets-api daily tend to prefer the native setup because they don't have to deal with the abstraction of docker-compose while those who would to spend less time on getting started prefer the docker setup. Docker is also useful when it's necessary to have a setup as close to production as possible.
 
-To start, fetch this code:
+[native setup instructions](docs/setup/native.md) (OSX/Ubunty)
+[docker setup instructions](docs/setup/docker.md)
```

Comment:
Not sure why, but these didn't appear on new lines for me when I previewed. Even so, some bullets points look nicer imo.
```suggestion
- [native setup instructions](docs/setup/native.md) (OSX/Ubunty)
- [docker setup instructions](docs/setup/docker.md)
```